documentation/src/docs/beginner_dev_guide.md

```diff
@@ -0,0 +1,199 @@
+
+# 2. Goal
+In this guide, we will develop a straightforward application that creates a Stream on Pravega and writes an event into the Stream and reads back from it.
+We recommend that you follow the instructions from [Bootstrapping project](#4-Bootstrapping-the-Project) onwards to create the application step by step.
+However, you can go straight to the completed example at [Pravega-samples-repo](https://github.com/pravega/pravega-samples).
+
+<details open>
+<summary>Command to clone the pravega-samples repo:</summary>
+<p>
+
+```console
+$ git clone https://github.com/pravega/pravega-samples.git
+```
+
+</p>
+</details>  
+
+
+The solution is located in the pravega-client-examples [directory]( https://github.com/pravega/pravega-samples/tree/master/pravega-client-examples/src/main/java/io/pravega/example/gettingstarted ).
+
+# 3. Starting Pravega Standalone
+In standalone mode, the Pravega server is accessible from clients through the `localhost` interface only. Controller REST APIs, however, are accessible from remote hosts/machines.
+You can launch a standalone mode server using either of the following options:
+
+1. From source code:
+
+<details open>
+<summary>Commands to start standalone from source code:</summary>
+<p>
+
+Checkout the source code.
+```console
+$ git clone https://github.com/pravega/pravega.git
+$ cd pravega
+```
+Build the Pravega standalone mode distribution.
+
+```console
+$ ./gradlew startStandalone
+```
+
+</p>
+</details>
+
+2. From installation package:
+
+<details open>
+<summary>Commands to start standalone from installation package:</summary>
+<p>
+
+Download the Pravega release from the [GitHub Releases](https://github.com/pravega/pravega/releases).
+
+```console
+$ tar xfvz pravega-<version>.tgz
+```
+Download and extract either tarball or zip files. Follow the instructions provided for the tar files (same can be applied for zip file) to launch all the components of Pravega on your local machine.
+
+Run Pravega Standalone:
+
+```console
+$ pravega-<version>/bin/pravega-standalone
+```
+
+</p>
+</details>  
+
+
+# 4. Bootstrapping the Project
+
+The easiest way to bootstrap a sample application against Pravega is to run the following command in a folder of your choice.
+```console
+$ gradle init --type java-application
+```
+Add the below snippet to dependencies section of build.gradle in the app directory.
+```groovy
+// https://mvnrepository.com/artifact/io.pravega/pravega-client
+implementation group: 'io.pravega', name: 'pravega-client', version: '0.9.0'
+```
+Invoke `gradle run` to run the project.
+
+
+<details open>
+<summary>Expected output:</summary>
+<p>
+
+```console
+$ gradle run
+
+> Task :app:run
+Hello World!
+
+BUILD SUCCESSFUL in 890ms
+2 actionable tasks: 2 executed
+```
+
+</p>
+</details>
+
+## 4.1 Create a Pravega Stream
+
+Let’s first get to know Pravega’s client APIs by creating a stream with a fixed scaling policy of 1 segment. We’ll need a StreamConfiguration to define this.
```

Comment:
`Let’s`, `We’ll` -> looks like the `'` character is not right. 
In the paragraph below, it looks right in `Pravega let's`.

Reply:
fixed it :)